qCC/ccCommandLineParser.cpp

```diff
@@ -544,7 +567,7 @@ bool ccCommandLineParser::saveMeshes(QString suffix/*=QString()*/, bool allAtOnc
 				CommandSave::SetFileDesc(desc, *allAtOnceFileName);
 			}
 
-			QString errorStr = exportEntity(desc, suffix, nullptr, false);
+			QString errorStr = exportEntity(desc, suffix, nullptr, ExportOption::ForceMesh);
```

Comment:
I might have missed it - in the old code is forcing it to mesh the default?

Reply:
The old code relied on it being not a cloud to use mesh, now it's just explicit